aws_lambda_powertools/metrics/base.py

```diff
@@ -358,6 +393,39 @@ def decorate(event, context):
 
         return decorate
 
+    def _extract_metric_resolution_value(self, resolution: Union[int, MetricResolution]) -> int:
+        """Return metric value from metric unit whether that's str or MetricResolution enum
+
+        Parameters
+        ----------
+        unit : Union[int, MetricResolution]
+            Metric resolution
+
+        Returns
+        -------
+        int
+            Metric resolution value must be 1 or 60
+
+        Raises
+        ------
+        MetricResolutionError
+            When metric resolution is not supported by CloudWatch
+        """
+
+        if isinstance(resolution, int):
+            if resolution in self._metric_resolution_options:
```

Comment:
suggestion: try renaming to `_metric_resolution_valid_options` or something along these lines so it's quicker to read code for the intent you have at hand (validate whether resolution falls within the valid options.)

Reply:
Renamed